pinot-azure-filesystem/src/main/java/com/linkedin/pinot/filesystem/AzurePinotFS.java

```diff
@@ -99,8 +99,7 @@ public boolean move(URI srcUri, URI dstUri, boolean overwrite) throws IOExceptio
     if (exists(dstUri) && !overwrite) {
       return false;
     }
-    //rename the file
-    return _adlStoreClient.rename(srcUri.getPath(), dstUri.getPath());
+    return copy(srcUri, dstUri) && delete(srcUri, true);
```

Comment:
Could you elaborate how the rename() api is different from copy() && delete()

Reply:
`rename()` with `overwrite` param set to `true` should give the same functionality.